front-end/src/components/player-page/stats/stats-generators/DiffStatGenerator.ts

```diff
@@ -0,0 +1,69 @@
+import {BaseStatGenerator} from "./BaseStatsGenerator";
+import PlayerStatModel from "../../../../../../Common/models/playerstat.model";
+
+export enum DiffEnum {
+    CS = "CS",
+    GOLD = "GOLD",
+    XP = "XP",
+}
+
+const goldDiffResolver: Record<15 | 25, (x: PlayerStatModel) => number> = {
+    15 : (model: PlayerStatModel) => {return model.goldDiff15},
```

Comment:
fyi you can remove the {} and return. So it would just be `(model: PlayerStatModel) => model.goldDiff15`

Reply:
yea i kinda did it for 1 then i was copy pasting. Will fix